client/utils/collections/api.ts

```diff
@@ -1,6 +1,17 @@
 import { apiFetch } from 'client/utils/apiFetch';
 
 const collectionPubsRoot = '/api/collectionPubs';
+const collection = '/api/collections';
+
+export const createTagCollection = ({ title, communityId }) =>
+	apiFetch(collection, {
+		method: 'POST',
+		body: JSON.stringify({
```

Comment:
I wonder if we should pass `isPublic: true` here to make these public by default — maybe create a parameter for this too. Paging @gabestein

Reply:
yup, I think that's right.

Reply:
@idreyn what do you mean by create a parameter? 

is this what you are getting at? Why if it is always true?
`export const createTagCollection = ({ title, communityId, isPublic }) =>
	apiFetch(collection, {
		method: 'POST',
		body: JSON.stringify({
			communityId,
			title,
			kind: 'tag',
			isPublic,
		}),
	});`

Reply:
This is what I mean — and because it may be useful to let future callers of the function decide whether the tag should be public or not. You could argue that this is premature! But I do think it's best for functions to default to a less surprising/destructive action (in this case, creating a Collection that's initially public) and let callers opt-in to the more surprising behavior. So the function signature would be

```ts
({ title, communityId, isPublic = false }) => ...
```